### PR TITLE
Update Go to 1.19.4

### PR DESCRIPTION
**Pull request by embik**
**What this PR does / why we need it**:
What it says on the title. Updates to Go 1.19.4 build images.

**Which issue(s) this PR fixes**:

Fixes #

**What type of PR is this?**
/kind feature
/kind chore

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change? Then add your Release Note here**:

```release-note
Update Go to version 1.19.4
```

**Documentation**:

```documentation
NONE
```

### CODE REVIEW

**Review by ahmedwaleedmalik — APPROVED** [review]
/approve ⚡

**Comment by embik** [comment]
/hold

Waiting for postsubmit job updates before merging this one.

**Pull request merged**
